jmix-templates/content/flowui/master-detail/descriptor.xml

```diff
@@ -90,7 +90,7 @@ def optionsDatasourceXml = api.processSnippet('optionsDsSource.xml',
         <action id="discardAction" type="lookup_discard"/>
     </actions>
     <layout padding="false">
-        <formLayout classNames="master-detail-layout" height="100%">
+        <formLayout id="masterDetailForm" classNames="master-detail-layout" height="100%">
```

Comment:
`masterDetail` is the name of view template. the `formLayout` is just a form in the detail part of the view. Give it the same id as in the regular detail view

Reply:
The descriptor already contains formLayout with id is equals "form". I suggest one of the following options:
1. masterForm and form
2. form and detailsForm
3. masterForm and detailsForm